libs/clients/cms/src/lib/cms.config.ts

```diff
@@ -18,14 +18,7 @@ export const CmsConfig = defineConfig({
       env.optional('GRAPHQL_API_ENDPOINT') ??
       'https://graphql.contentful.com/content/v1/spaces/8k0h54kbe6bj/environments/master',
     redis: {
-      nodes: env.requiredJSON('REDIS_URL_NODE_01', [
-        'localhost:7000',
-        'localhost:7001',
-        'localhost:7002',
-        'localhost:7003',
-        'localhost:7004',
-        'localhost:7005',
-      ]),
+      nodes: env.requiredJSON('REDIS_URL_NODE_01', []),
```

Comment:
Why clear the devFallback values? This might break current developer workflows (run docker compose, start app normally).

Reply:
Fair point, I reverted this change

Reply:
Yeah, this was leftovers from testing locally, and I couldn't get localstack to behave :sweat_smile: